src/globus_cli/commands/group/member/add.py

```diff
@@ -0,0 +1,45 @@
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import IdentityType, ParsedIdentity, command
+from globus_cli.termio import FORMAT_TEXT_RECORD, formatted_print
+from globus_cli.types import FIELD_LIST_T
+
+
+def _format_added_user_id(x: dict):
+    return x["add"][0]["identity_id"]
+
+
+def _format_added_user_username(x: dict):
+    return x["add"][0]["username"]
+
+
+def _format_group_id(x: dict):
+    return x["add"][0]["group_id"]
+
+
+ADD_USER_FIELDS: FIELD_LIST_T = [
+    ("Group ID", _format_group_id),
+    ("Added User ID", _format_added_user_id),
+    ("Added User Username", _format_added_user_username),
+]
+
+
+@command("add", short_help="Add one or more members to a group")
+@click.argument("group", type=str)
+@click.argument("user", type=IdentityType())
+@LoginManager.requires_login(LoginManager.GROUPS_RS)
+def member_add(group: str, user: ParsedIdentity, login_manager):
+    """Add one or more members to a group"""
+    auth_client = login_manager.get_auth_client()
+    groups_client = login_manager.get_groups_client()
+    identity_id = auth_client.maybe_lookup_identity_id(user.value)
+    if not identity_id:
+        raise click.UsageError(f"Couldn't determine identity from user value: {user}")
+    actions = {"add": [{"identity_id": identity_id}]}
+    response = groups_client.batch_membership_action(group, actions)
+    formatted_print(
+        response,
+        text_format=FORMAT_TEXT_RECORD,
+        fields=ADD_USER_FIELDS,
+    )
```

Comment:
I believe that this `formatted_print` call is the correct place to handle the case where the user is already a member of the group.

Here's an example of the full response data:
```json
{
  "add": [],
  "errors": {
    "add": [
      {
        "code": "ALREADY_ACTIVE",
        "detail": "The identity is already an active member of the group.",
        "identity_id": "c699d42e-d274-11e5-bf75-1fc5bf53bb24"
      }
    ]
  }
}
```

We may want more sophisticated behavior in the future, but for now I think the correct approach is for the CLI to treat this type of response as an error condition. We could raise a `ValueError`, or a custom error class -- either would be acceptable. That will get us a relatively clean `exit(1)` plus a message.

It's okay to try to unpack the `"detail"` and `"code"` fields, but please assume if you do so that the fields may not be present. I don't know if it's possible in fact for a response to come back which doesn't populate `"add"` in the main response body and also doesn't populate `"errors"`, but it's at least possible in theory.